op-node/p2p/sync.go

```diff
@@ -390,6 +396,16 @@ func (s *SyncClient) onRangeRequest(ctx context.Context, req rangeRequest) {
 		}
 	}
 
+	// Create shared randomReqId so associated peerRequests can all be cancelled by setting a single flag
```

Comment:
Could we use an atomic counter here instead of random data?

Reply:
Yes that would be easier than generating a random number. The tradeoff is that the `reqId` will overlap because the counter will reset every time the node restarts, but this is probably ok. Will make that change